scripts/build_yara

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Build yara for Heroku.
+# This script is designed to run in a Heroku Stack Docker
+# image. More information on the Heroku Stack can be found
+# at https://devcenter.heroku.com/articles/stack
+
+YARA_VERSION=${YARA_VERSION-4.3.2}
```

Comment:
Noting that this is a potentially breaking change because it's a major version upgrade. That's not to say we shouldn't upgrade.

We may want to later use tags to pin to specific releases.

Reply:
oh yes, I will try to compile 3.5 to not break too much

Reply:
Honestly, it's broken now so it will be easier to upgrade now than later. It's also fine not doing it but based on our use case, I would probably go for it.